edit/edit.js

```diff
@@ -295,13 +295,13 @@ function acmeEventListener(event) {
     return;
   }
   let value = el.type == 'checkbox' ? el.checked : el.value;
-  let url;
-  const themeLink = document.getElementById('cm-theme');
   switch (option) {
     case 'tabSize':
       CodeMirror.setOption('indentUnit', Number(value));
       break;
+    /* eslint-disable no-case-declarations */
```

Comment:
Hey, what do you think of an alternative "hack", which is probably better than tweaking eslint:

```
case 'theme': {
...........
}
case ......
```